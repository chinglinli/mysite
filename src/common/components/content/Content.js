import React from 'react'
import styles from './Content.scss'

export default class Content extends React.Component {
  render () {
    return (
      <div className={styles.root}>
        {this.props.children}
      </div>
    )
  }
}
