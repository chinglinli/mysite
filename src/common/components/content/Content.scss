@import '../../css/colors.scss';

.root {
  background: $sub-dominant-color;
  padding: 0.3rem;
}
