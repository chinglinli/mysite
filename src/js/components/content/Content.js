import * as React from 'react';

import styles from './Content.css';


export default class Content extends React.Component {
    render() {
        return (
            <div className={styles.root}>
                content
            </div>
        )
    }
}
