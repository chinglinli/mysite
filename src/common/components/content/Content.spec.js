import React from 'react'
import renderer from 'react-test-renderer'

import Content from './Content'


test('It should render Content component correctly', () => {
  const component = renderer.create(
    <Content>
      <div>今天天氣不錯</div>
    </Content>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
