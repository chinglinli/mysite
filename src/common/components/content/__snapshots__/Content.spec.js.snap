// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render Content component correctly 1`] = `
<div
  className="root"
>
  <div>
    今天天氣不錯
  </div>
</div>
`;
